### PR TITLE
add ænalytics & ArkaneNetwork

### PR DESCRIPTION
**Pull request by marc0olo**

### CODE REVIEW

**Comment by marc0olo** [comment]
@venimus can you merge this one? :-)

File: index.md [hunk 1]
```diff
@@ -324,12 +324,16 @@ Programming libraries are available. Any Javascript developer can build a simple
...
+    - **ænalytics**
+      - Mainnet [https://aenalytics.org](https://aenalytics.org)
+      - Testnet [https://testnet.aenalytics.org](https://testnet.aenalytics.org)
   - #### Wallets
     - Base æpp (mobile) [https://base.aepps.com](https://base.aepps.com)
     - Ledger Hardware Wallet (desktop) [https://base.aepps.com](https://base.aepps.com)
     - Trust Wallet (Binance)  [https://trustwallet.com](https://trustwallet.com)
     - Waellet (browser extension) [https://waellet.com](https://waellet.com)
     - AirGap (iOS/Android) [https://airgap.it](https://airgap.it)
+    - ArkaneNetwork [https://arkane.network](https://arkane.network)
```
Comment (venimus) [inline index.md:336]: I believe there were some issues with the wallet that is was left for review

```suggestion
```

**Comment by marc0olo** [comment]
@venimus may I know what's the issue with ArkaneNetwork? otherwise I cannot say whether this is solved or not

**Comment by venimus** [comment]
I guess my concerns were resolved so I can merge that

**Pull request merged**